libs/sdk-model/src/execution/filter/index.ts

```diff
@@ -187,6 +187,23 @@ export interface IMeasureValueFilter {
     };
 }
 
+/**
+ * @public
+ */
+export type RankingFilterOperator = "TOP" | "BOTTOM";
+
+/**
+ * @public
+ */
+export interface IRankingFilter {
+    rankingFilter: {
+        measures: (UriRef | LocalIdRef)[];
+        attributes?: (UriRef | LocalIdRef)[];
```

Comment:
heads up, the sdk-model must not use UriRef directly. Instead where reference to other objects is needed, use ObjRef (which is either URI or ID). You can use `ObjRefInScope` type. Which is union of all Id, Uri and LocalId refs.

Reply:
In this case it should be `ObjRefInScope` as all three are possible to execute. Uri and LocalID is only for visualization object saving (in its Ref form).

Reply:
Noted. In that case, measure value filter is also wrong, and does not support identifier in execution interface

Reply:
indeed. I see we used to have MVF as ObjRefInScope but then due to 'something' we switched to Uri | Local .. i'm trying to find out why that happened

Reply:
@yanlipnican i found out why it is like this. i'm going to make changes to MVF and restore the previous state.